components/dashboard/sections/collectives/FreezeAccountModal.tsx

```diff
@@ -0,0 +1,410 @@
+  onSuccess?: () => void;
+  onClose: () => void;
+} & Omit<React.ComponentProps<typeof StyledModal>, 'children'>) => {
+  const intl = useIntl();
+  const { toast } = useToast();
+  const [useMessageForAccountAdmins, setUseMessageForAccountAdmins] = useState(false);
+  const [messageForAccountAdmins, _setMessageForAccountAdmins] = useState('');
+  const [pauseContributions, setPauseContributions] = useState(true);
+  const [hasConfirmed, setHasConfirmed] = useState(false);
+  const [submitAction, setSubmitAction] = useState<ModalState | null>(null);
+  const { data, loading, error } = useQuery(freezeAccountModalQuery, {
+    variables: { accountId: collective.id },
+    context: API_V2_CONTEXT,
+  });
+  const [editAccountFreezeStatus, { loading: submitting }] = useMutation(editAccountFreezeStatusMutation, {
+    context: API_V2_CONTEXT,
+  });
+  const setMessageForAccountAdminsFromInputEvent = React.useCallback(
+    e => _setMessageForAccountAdmins(e.target.value),
+    [],
+  );
+
+  const defaultState = collective.isFrozen ? 'UNFREEZE' : 'FREEZE';
+  const modalState = getModalState(data?.account, submitAction, defaultState);
+  const isUnfreezing = modalState === 'UNFREEZE';
+  const contributionsSummary = getRecurringContributionsCountSummary(data?.account);
+  return (
+    <StyledModal maxWidth={600} trapFocus onClose={onClose} {...props}>
+      <CollectiveModalHeader
+        mb={3}
+        collective={collective}
+        preText={
+          isUnfreezing
+            ? intl.formatMessage({ defaultMessage: `Unfreeze`, id: '5SBeLS' })
+            : intl.formatMessage({ defaultMessage: `Freeze`, id: 'Account.Freeze' })
+        }
+      />
+
+      <ModalBody mb={0}>
+        {loading ? (
+          <Skeleton className="h-40" />
+        ) : error ? (
+          <MessageBoxGraphqlError error={error} mb={2} />
+        ) : isUnfreezing ? (
+          <div>
+            <h2 className="mb-4 text-xl font-bold">
+              <FormattedMessage defaultMessage="Are you sure want to unfreeze this collective?" id="OX8+5o" />
+            </h2>
+
+            <p className="mb-6 text-sm">
+              <FormattedMessage
+                defaultMessage="Unfreezing the collective means they will regain full access to the platform."
+                id="dMaivT"
+              />
+              <br />
+              <br />
+              <FormattedMessage
+                defaultMessage="This collective (and all its related Projects & Events) will now have access to accept funds, pay out expenses, post updates, create new Events or Projects."
+                id="1Whmi8"
+              />{' '}
+              <FormattedMessage defaultMessage="Their existing recurring contributions will be resumed." id="VxS/8s" />
+            </p>
+            <Collapsible open={useMessageForAccountAdmins}>
+              <CollapsibleTrigger asChild>
+                <div className="mb-4 flex items-center justify-between rounded-md border border-gray-200 p-4">
```

Comment:
Default border color is `slate-200`, so we can remove the color from here:
```suggestion
                <div className="mb-4 flex items-center justify-between rounded-md border p-4">
```